packages/core/src/browser/addEventListener.ts

```diff
@@ -100,3 +98,28 @@ export function addEventListeners<E extends Event>(
     stop,
   }
 }
+
+let untrustedEventsReported: Set<string> | undefined
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** from the naming, I would have expected a boolean indicating if untrusted events are reported or not‏
```suggestion
let reportedUntrustedEventTypes: Set<string> | undefined
```